packages/code-du-travail-api/src/server/routes/__tests__/__snapshots__/highlights.test.js.snap

```diff
@@ -3,17 +3,7 @@
 exports[`return highlights matching the slug, correctly hydrated 1`] = `
 Array [
   Object {
-    "breadcrumbs": Array [
```

Comment:
is it ok that breadcrumbs disappear ?

Reply:
nope, there's probably an issue with the updated data, hence my "not convinced" commit message ;)
need to investigate